simplq/src/components/pages/Admin/index.jsx

```diff
@@ -138,7 +138,9 @@ export default (props) => {
         onRequestClose={closeTour}
         onAfterOpen={disableScroll}
         onBeforeClose={enableScroll}
-      />
+        getCurrentStep={stepChange}
+        >
+        </Tour>
```

Comment:
@promarcussmith I am wondering if it's feasible to keep the `<Tour />` and it's logic together inside `common/Tour/Tour.jsx` and just import and use it here.

`Admin/TourSteps.jsx` would just have just the tour steps defined.
All other common logic and the Tour component would go into `common/Tour` folder

and then we would use common/Tour like this in this file:
```
import Tour from 'common/Tour';
import { getToursteps } from './TourSteps';
...
export default (props) => {
  ...
  return (
    ...
    <Tour steps={tourSteps} />
    ...
  );
```

Reply:
Sure, can do this.

Reply:
Sure, let me see what changes need to be done here.

Reply:
Done